### PR TITLE
fix: spine resource multi name error

### PR DESCRIPTION
**Pull request by fanmingfei**
### Please check if the PR fulfills these requirements

- [ ] The commit message follows our [guidelines](https://github.com/eva-engine/eva.js/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Other information:

### CODE REVIEW

File: packages/plugin-renderer-spine/lib/SpineSystem.ts [hunk 1]
```diff
@@ -78,7 +78,8 @@ export default class SpineSystem extends Renderer {
   async add(changed: ComponentChanged, count?: number) {
     const component = changed.component as Spine;
     clearTimeout(component.addHandler);
-    const spineData = await getSpineData(component.resource);
+    const res = await resource.getResource(component.resource);
```
Comment (terrykingcha) [inline packages/plugin-renderer-spine/lib/SpineSystem.ts:81]: 这里不需要try/catch？

Comment (terrykingcha) [inline packages/plugin-renderer-spine/lib/SpineSystem.ts:81]: 这里不需要try/catch？

File: packages/plugin-renderer-spine/lib/SpineSystem.ts [hunk 2]
```diff
@@ -166,10 +166,8 @@ export default class SpineSystem extends Renderer {
 
     if (component.armature) {
       component.armature.destroy({ children: true });
-
-      if (component.usingResource) {
-        releaseSpineData(component.usingResource);
-      }
+      const res = await resource.getResource(component.resource)
```
Comment (terrykingcha) [inline packages/plugin-renderer-spine/lib/SpineSystem.ts:169]: try/catch？

File: packages/plugin-renderer-spine38/lib/SpineSystem.ts [hunk 3]
```diff
@@ -78,7 +78,8 @@ export default class SpineSystem extends Renderer {
   async add(changed: ComponentChanged, count?: number) {
     const component = changed.component as Spine;
     clearTimeout(component.addHandler);
-    const spineData = await getSpineData(component.resource);
+    const res = await resource.getResource(component.resource);
```
Comment (terrykingcha) [inline packages/plugin-renderer-spine38/lib/SpineSystem.ts:81]: try/catch

File: packages/plugin-renderer-spine38/lib/SpineSystem.ts [hunk 4]
```diff
@@ -166,10 +166,8 @@ export default class SpineSystem extends Renderer {
 
     if (component.armature) {
       component.armature.destroy({ children: true });
-
-      if (component.usingResource) {
-        releaseSpineData(component.usingResource);
-      }
+      const res = await resource.getResource(component.resource)
```
Comment (terrykingcha) [inline packages/plugin-renderer-spine38/lib/SpineSystem.ts:169]: try/catch？

File: packages/plugin-renderer-spine38/lib/SpineSystem.ts (hunk 3 above)
Comment (fanmingfei) [reply packages/plugin-renderer-spine38/lib/SpineSystem.ts:81]: 不需要后面有兼容

File: packages/plugin-renderer-spine38/lib/SpineSystem.ts (hunk 4 above)
Comment (fanmingfei) [reply packages/plugin-renderer-spine38/lib/SpineSystem.ts:169]: 不需要后面有兼容

File: packages/plugin-renderer-spine/lib/SpineSystem.ts (hunk 2 above)
Comment (fanmingfei) [reply packages/plugin-renderer-spine/lib/SpineSystem.ts:169]: 不需要后面有兼容

File: packages/plugin-renderer-spine/lib/SpineSystem.ts (hunk 1 above)
Comment (fanmingfei) [reply packages/plugin-renderer-spine/lib/SpineSystem.ts:81]: 不需要后面有兼容

**Pull request merged**